nautobot_netbox_importer/summary.py

```diff
@@ -38,8 +38,7 @@ class FieldSummary(NamedTuple):
     nautobot_can_import: Optional[bool]
     importer: Optional[str]
     definition: Union[str, bool, int, float, None]
-    from_data: Optional[bool]
-    is_custom: Optional[bool]
+    sources: List[str]
```

Comment:
I may be missing somethign, but I don't see this attribute initialized, just used

Reply:
Initialized here:

https://github.com/nautobot/nautobot-app-netbox-importer/pull/135/files#diff-fe77f230ddce96ba9a9d7b349e5e7acfe3c00818cd1818c543d2c1f29504d39dR713